client/src/components/hooks/useLocalStorage.js

```diff
@@ -0,0 +1,28 @@
+import { useState } from "react"
+
+export const useLocalStorage = (key, initialValue = "") => {
+
+	const [storedValue, setStoredValue] = useState(() => {
+
+		try {
+			const currentLocalValue = window.localStorage.getItem(key)
+
+			return currentLocalValue ? JSON.parse(currentLocalValue) : initialValue
+		} catch (error) {
+			console.log(error);
```

Comment:
Always remember to remove redundant codes from PRs

Reply:
What do you mean by redundant code?

Reply:
The console.log(error) wasn't meant to be a redundant code. I had to do something with the error I was catching and I decided to log it in the console.